docs_src/examples/Ch-ExamplesVirtualDeviceService.md

```diff
@@ -88,79 +56,99 @@ curl -X GET localhost:48082/api/v1/device/1bd5d4c3-9d43-42f2-8c4a-f32f5999edf7/c
   "EncodedEvent": null
 }
 ```
+
 ### PUT command example - Assign a value to a resource
+The virtual devices managed by the virtual device can also be actuated.  The virtual device can be told to enable or disable random number generation.  When disabled, the virtual device services can be told what value to respond with for all `GET` operations.  When setting the fixed value, the value must be valid for the data type of the virtual device. For example, the minimum value of Int8 cannot be less than -128 and the maximum value cannot be greater than 127.
+
+Below is example actuation of one of the virtual devices.  In this example, it sets the fixed 'GET` return value to 123 and turns of random generation.
```

Comment:
typos:
- 'GET` -> The first single quotation mark should be replaced with a grave accent mark.
- turns of -> turns off